ios/ImagePickerManager.m

```diff
@@ -89,17 +89,14 @@ - (void)imagePickerController:(UIImagePickerController *)picker didFinishPicking
 
         if ([info[UIImagePickerControllerMediaType] isEqualToString:(NSString *) kUTTypeImage]) {
             UIImage *image = [ImagePickerManager getUIImageFromInfo:info];
-            [assets addObject:[self mapImageToAsset:image data:nil]];
+            [assets addObject:[self mapImageToAsset:image data:[NSData dataWithContentsOfURL:[ImagePickerManager getNSURLFromInfo:info]]]];
```

Comment:
`nil` is passed in intentionally. the key getNSURLFromInfo tries to get doesnt exist. see my comment (and debugger output): https://github.com/react-native-image-picker/react-native-image-picker/pull/1706/files#r633383052

Reply:
Did u check in ios13, because this value cannot be nil according to ios docs

Reply:
the screenshot is from a ios 14.5 sim, although i cannot reproduce it on current branch. so seems all good to me. (it will work regardless)